src/main/java/com/nekomaster1000/infernalexp/blocks/PlantedQuartzBlock.java

```diff
@@ -0,0 +1,113 @@
+package com.nekomaster1000.infernalexp.blocks;
+
+import com.nekomaster1000.infernalexp.init.IEBlocks;
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.Blocks;
+import net.minecraft.item.Item;
+import net.minecraft.item.Items;
+import net.minecraft.state.StateContainer;
+import net.minecraft.state.properties.AttachFace;
+import net.minecraft.util.Direction;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.shapes.ISelectionContext;
+import net.minecraft.util.math.shapes.VoxelShape;
+import net.minecraft.world.IBlockReader;
+import net.minecraft.world.IWorldReader;
+import net.minecraft.world.World;
+
+import javax.annotation.CheckForNull;
+
+public class PlantedQuartzBlock extends HorizontalBushBlock {
+    protected static final VoxelShape FLOOR_SHAPE = Block.makeCuboidShape(2.0D, 0.0D, 2.0D, 14.0D, 12.0D, 14.0D);
+    protected static final VoxelShape CEILING_SHAPE = Block.makeCuboidShape(2.0D, 4.0D, 2.0D, 14.0D, 16.0D, 14.0D);
+    protected static final VoxelShape WALL_SHAPE_NORTH = Block.makeCuboidShape(2.0D, 2.0D, 4.0D, 14.0D, 14.0D, 16.0D);
+    protected static final VoxelShape WALL_SHAPE_SOUTH = Block.makeCuboidShape(2.0D, 2.0D, 0.0D, 14.0D, 14.0D, 12.0D);
+    protected static final VoxelShape WALL_SHAPE_EAST = Block.makeCuboidShape(0.0D, 2.0D, 2.0D, 12.0D, 14.0D, 14.0D);
+    protected static final VoxelShape WALL_SHAPE_WEST = Block.makeCuboidShape(4.0D, 2.0D, 2.0D, 16.0D, 14.0D, 14.0D);
+
+    public PlantedQuartzBlock(Properties builder) {
+        super(builder);
+        this.setDefaultState(this.getDefaultState().with(FACE, AttachFace.FLOOR).with(HORIZONTAL_FACING, Direction.NORTH));
+    }
+
+    @Override
+    protected boolean isValidGround(BlockState state, IBlockReader worldIn, BlockPos pos) {
+        return state.matchesBlock(Blocks.QUARTZ_BLOCK) || state.matchesBlock(Blocks.CHISELED_QUARTZ_BLOCK)
+            || state.matchesBlock(Blocks.SMOOTH_QUARTZ) || state.matchesBlock(Blocks.QUARTZ_BRICKS)
+            || state.matchesBlock(Blocks.QUARTZ_PILLAR) || state.matchesBlock(Blocks.NETHER_QUARTZ_ORE)
+            || state.matchesBlock(Blocks.SOUL_SAND) || state.matchesBlock(Blocks.SOUL_SOIL)
+            || state.matchesBlock(Blocks.NETHERRACK);
+    }
```

Comment:
convert this to a tag

Reply:
This is what the BuriedBoneBlock does. Should I change that one too?

Reply:
yes